routes/actions.js

```diff
@@ -347,4 +348,10 @@ function sendRegistrationForm(req, res, formInfo) {
   });
 }
 
+function returnToPath(reqOrig, res) {
+    let reqUrl = reqOrig.originalUrl;
+    let query = url.parse(reqUrl, true).query.returnTo;
```

Comment:
Is there a reason you are not just using req.query.returnTo here?

Reply:
I thought req.query was incompatible with post requests due to some earlier issues. It's working now so I'll rewrite it to use that.